### PR TITLE
Added color themes to frontend

### PR DESCRIPTION
**Pull request by ThorbenStaufer**
Added the two color themes discussed in the focus group meeting to the front end and applied them to the design.

## Basics

- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.
- [ ] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I mentioned [every code or binary](/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is conforming to [our Documentation Guidelines](/doc/documentation.md)
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to our Coding Guidelines
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

### CODE REVIEW

File: frontend/src/components/Button/SimpleButton.tsx [hunk 1]
```diff
@@ -3,8 +3,10 @@ interface SimpleButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement
 }
```
Comment (badnames) [inline frontend/src/components/Button/SimpleButton.tsx:4]: Consider adding a short comment (or a new file in the doc folder), outlining the intended use of these buttons.

**Review by badnames — APPROVED** [review]
Well done!
Only one minor nitpick.

**Review by buenaflor — APPROVED** [review]
nice!

**Comment by markus2330** [comment]
Thank you all! :heart:

**Pull request merged**